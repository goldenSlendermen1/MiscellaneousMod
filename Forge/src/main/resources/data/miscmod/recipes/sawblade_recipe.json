{
  "type": "minecraft:crafting_shaped",
  "group": "ingrediants",
  "category": "misc",
  "pattern": [
    " a ",
    "aaa",
    " a "
  ],
  "key": {
    "a": {
      "item": "minecraft:iron_ingot"
    }
  },
  "result": {
    "item": "miscmod:sawblade",
    "count": 1
  }
}