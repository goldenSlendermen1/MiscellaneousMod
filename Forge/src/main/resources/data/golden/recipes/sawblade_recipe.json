{
  "type": "minecraft:crafting_shaped",
  "group": "ingrediants",
  "category": "misc",
  "pattern": [
    " a ",
    "aaa",
    " a "
  ],
  "key": {
    "a": {
      "item": "minecraft:iron_ingot"
    }
  },
  "result": {
    "item": "golden:sawblade",
    "count": 1
  }
}